tests/unit/samplegen/test_integration.py

```diff
@@ -257,6 +273,63 @@ def test_generate_sample_config_fpaths_bad_contents_no_samples(fs):
     )
 
 
+def test_generate_sample_config_partial_config(fs):
+    expected_path = 'sample.yaml'
+    fs.create_file(
+        expected_path,
+        contents=dedent(
+            '''
+            ---
+            # Note: this one is NOT a valid config
```

Comment:
mention the `Pronto` typo to help humans

Reply:
Done